docs/plugins/discovery-ec2.asciidoc

```diff
@@ -177,9 +177,7 @@ Management Console. It should look similar to this.
 ===== Filtering by Tags
 
 The ec2 discovery can also filter machines to include in the cluster based on tags (and not just groups). The settings
-to use include the `discovery.ec2.tag.` prefix. For example, setting `discovery.ec2.tag.stage` to `dev` will only
-filter instances with a tag key set to `stage`, and a value of `dev`. Several tags set will require all of those tags
-to be set for the instance to be included.
+to use include the `discovery.ec2.tag.` prefix. For example, if you defined a tag `stage` in ec2 and set it to `dev`, setting `discovery.ec2.tag.stage` to `dev` will only filter instances with a tag key set to `stage`, and a value of `dev`. Several tags set will require all of those tags to be set for the instance to be included. http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/Using_Tags.html[Read more about EC2 Tags].
```

Comment:
In general, we try to avoid external links in the docs because they break frequently. Is the link truly needed here?

Reply:
I wanted to enforce the idea that the tags should be defined in EC2 machines to be able to be used for filtering, as it's not obvious. And, since the link was already used a bit further up in the same page, I thought it was a good way for the mentioned enforcement.

Reply:
The verbiage is fine, it's only the link that I object to. Let's not add it here and we can consider removing it from the other location in a follow-up. Additionally, in your change please capitalize "ec2" to "EC2" (and let us not worry about the other places in the page where this is not done, that should be cleaned up in a separate follow-up).